src/core/sys/freebsd/sys/event.d

```diff
@@ -15,6 +15,8 @@ module core.sys.freebsd.sys.event;
 
 version (FreeBSD):
 extern (C):
+nothrow:
+@nogc:
```

Comment:
With regards to `@nogc` I see your point, but `nothrow`? Is throwing from a D function called by C even defined behaviour?